src/commands/images/generateSbom.ts

```diff
@@ -0,0 +1,29 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See LICENSE.md in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { IActionContext, openReadOnlyJson } from '@microsoft/vscode-azext-utils';
+import * as vscode from 'vscode';
+import { ext } from '../../extensionVariables';
+import { ImageSbomItem } from '../../runtimes/docker';
+import { ImageTreeItem } from '../../tree/images/ImageTreeItem';
+
+export async function generateSbom(context: IActionContext, node?: ImageTreeItem): Promise<void> {
+    if (!node) {
+        await ext.imagesTree.refresh(context);
+        node = await ext.imagesTree.showTreeItemPicker<ImageTreeItem>(ImageTreeItem.contextValue, {
+            ...context,
+            noItemFoundErrorMessage: vscode.l10n.t('No images are available for generating sbom')
+        });
+    }
+
+    const generating: string = vscode.l10n.t('Generating sbom...');
```

Comment:
```suggestion
    const generating: string = vscode.l10n.t('Generating Software Bill of Materials...');
```

Reply:
In general, you should say "Software Bill of Materials" to the user, unless the space is really tight. In the latter case use "SBOM" (all caps).

Reply:
That makes sense